components/gantry/multiaxis/multiaxis.go

```diff
@@ -115,18 +116,28 @@ func (g *multiAxis) MoveToPosition(
 		return errors.Errorf("need position inputs for %v-axis gantry, have %v positions", len(g.subAxes), len(positions))
 	}
 
-	idx := 0
+	if len(positions) != len(g.lengthsMm) {
+		return errors.Errorf(
+			"number of input positions %v does not match total gantry axes length %v",
+			len(positions), len(g.lengthsMm),
+		)
+	}
+
+	jdx := 0
```

Comment:
Why did `idx` get renamed `jdx`? I'd prefer to go back to the old name, which is obviously short for "index." I don't know what `jdx` is short for.

Reply:
it's common in the non-software world to index using i, j, k, so i actually think in i-index, j-index, k-index.